packages/react-components/src/icons/github.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+/* istanbul ignore file */
+
+const githubIcon = (
+  <svg
+    width={28}
+    height={28}
+    viewBox="0 0 28 28"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <title>Github</title>
```

Comment:
```suggestion
    <title>GitHub</title>
```
(don't care about it in the identifiers since for us as programmers I guess it's one fixed term)